### PR TITLE
Feat/tutorial updates

### PR DESCRIPTION
**Pull request by rubychilds**
- re-organisation of files
- some improved documentations (more to come)
- some updated imagery throughout
- no longer hosting images on digital ocean
- pagination
- handling of image and asset directories
- tabs for different install instructions like docker

### CODE REVIEW

File: docs/how-to/docker.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Running Models Locally
+
+Here at Runway, we use [Docker](https://www.docker.com/) to help us run our AI models locally. You only need to use Docker, if you'd like to run some of the models locally. Here we discuss why we need to install Docker when you run Runway.
+
+![Select Model](assets/images/tutorials/tutorial_docker/docker-runway.png)
```
Comment (cvalenzuela) [inline docs/how-to/docker.md]:
```suggestion

```

File: docs/how-to/docker.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+
+## Which models can be run Locally?
+
+When you browse models, you can see in the model information, if it supports CPU and/or GPU. If the model supports CPU you can run on your local harddrive. Here we see the model Facial Landmarks is able to run locally.
```
Comment (cvalenzuela) [inline docs/how-to/docker.md]:
```suggestion
The model information regarding CPU and GPU support lives inside every model view page. If the model supports CPU and/or GPU will be described in the characteristics table. If the model supports CPU you can run it locally on your computer. For example, we can see that the model called Facial Landmarks is able to run locally:
```

File: docs/how-to/docker.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+![CPU model](assets/images/how-to/run-locally/model-cpu.png)
+
+## What is Docker?
+
+[Docker](https://www.docker.com/) is a tool which is designed to run, deploy and create applications, in a container. Containers are a way of making an environment similar to that of your computers Operating System (OS). This allows us to control what is installed within a container for your application, so we have only the things that are needed.
```
Comment (cvalenzuela) [inline docs/how-to/docker.md]:
```suggestion
[Docker](https://www.docker.com/) is a tool designed to run, deploy and create applications that live inside containers. Containers are a way of making closed and specific environments for every application. This allows to control what is installed within a container for your application, so you can have only the things that are needed.
```

File: docs/how-to/docker.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+Some of you may wonder if [Docker](https://www.docker.com/) is the same as a virtual machine (VM). Docker differs from a VM, as a VM will include a complete OS. The VM will act and run independently from your computer. But Docker in many ways, has a pick-and-choose kinder attitude and will use your computers resources, so it's less independent and arguably in many ways lighter to use.
```
Comment (cvalenzuela) [inline docs/how-to/docker.md]:
```suggestion
[Docker](https://www.docker.com/) is not the same as a virtual machine (VM). Docker differs from a VM, as a VM will include a complete OS. The VM will act and run independently from your computer. But Docker in many ways, has a pick-and-choose kinder attitude and will use your computer's resources, so it's less independent and arguably in many ways lighter to use.
```

File: docs/getting-started/model-101.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+# Models 101
+
+At the core of Runway, we have models. But what exactly is a model and how does it work? Here we discuss what a model is and why we need it within the Runway App.
```
Comment (brannondorsey) [inline docs/getting-started/model-101.md]: These first two sentences read a bit awkward to me, at least on the first pass through.

File: docs/getting-started/model-101.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+## What is a Model? How is a Model Created?
+
+We use the term model, as a high level representation of AI logic.  The model itself, contains a set of series and rules, that can be both decided and determined by humans and machine intelligence. For example, a model may itself learn how to recognise a face in a video or image. A researcher has given the model, images containing faces and over time, the model has learned what a face looks like. This is an example of a form of recognition. We often take recognition for granted, as we learnt how to recognise people and inanimate objects as we learned as a child. But imagine how tricky it is for a computer to learn and recognise!
```
Comment (brannondorsey) [inline docs/getting-started/model-101.md]:
```suggestion
We use the term model as a high level representation of AI logic.  Models contains a series parameters and soft rules that can be both decided by both humans and machine intelligence. For example, a model may itself learn how to recognize a face in a video or image. A researcher has given the model, images containing faces and over time, the model has learned what a face looks like. This is an example of a form of recognition. We often take recognition for granted, as we learned how to recognize people and inanimate objects as we learned as a child. But imagine how tricky it is for a computer to learn and recognize!
```

Comment (brannondorsey) [inline docs/getting-started/model-101.md]: I'm somewhat hesitant to define a model as containing `a series of rules`, as ML in general sort of pushing back against this idea.

File: docs/getting-started/model-101.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+When the researcher feeds the model by giving it some data such as images, text, video, this is what we call *training data*. When it's finished learning, the researcher then *tests* the model, to see if it's been able to learn. We see if the model has learned the data from if it's output is relevant, For example. can it recognise a face now it's seen so many images of faces?
```
Comment (brannondorsey) [inline docs/getting-started/model-101.md]:
```suggestion
When the researcher feeds the model by giving it some data such as images, text, video, this is what we call *training data*. When it's finished learning, the researcher then *tests* the model, to see if it's been able to learn. We see if the model has learned the data if its output is relevant. For example, can it recognize a face now that it's seen so many images of faces?
```

**Comment by brannondorsey** [comment]
In several places In my comments I suggested replacing the British English spelling of several words with the American English spelling of those words. Is that something we care to standardize or are we comfortable with a more fluid voice?

Posing that question ⬆️ openly.

File: docs/AI/creative-ai.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+# Creativity and AI
+
+Runway is a platform which enables creators to use the latest AI techniques. Here at Runway, we believe AI can be used as a tool in your creative process.
```
[the remark this answers is not in the document]
Comment (rubychilds) [reply docs/AI/creative-ai.md]: Completely agree! Aim to work on making these inntro's super strong annd informative about such differences

File: docs/README.md [hunk 9]
```diff
@@ -1,37 +1,30 @@
...
+First, let's get you [signed up!](https://runwayml.com/) Shortly upon signing up, you'll receive an email with a link for downloading the app.
 
 #### System requirements
 
 * Mac, Windows, or Linux operating system
 * More than 8GB of RAM
-* [Docker](https://www.docker.com/) installed.
-
-Please follow the installation guide [here](installation.md).
+* [Docker](https://www.docker.com/) installed - [see our guide to why we need Docker](/getting-started/docker)
```
[the remark this answers is not in the document]
Comment (rubychilds) [reply docs/README.md]: I agree. I place more emphasis on the run a model, and run a model locally - that's when Docker is introduced

**Comment by rubychilds** [comment]
@brannondorsey I'm thinking we should standardize to American English

File: docs/getting-started/model-101.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
+# Models 101
+
+At the core of Runway, we have machine learning models. But what do mean by the term model and how do we use a model? Here we discuss what a model is and why we need it within Runway.
+
+## What is a Model? How is a Model Created?
+
+We use the term model as a high level representation of AI logic.  Models contains a series parameters and soft rules that can be both decided by both humans and machine intelligence. For example, a model may itself learn how to recognize a face in a video or image. A researcher has given the model, images containing faces and over time, the model has learned what a face looks like. This is an example of a form of recognition. We often take recognition for granted, as we learned how to recognize people and inanimate objects as we learned as a child. But imagine how tricky it is for a computer to learn and recognize!
```
Comment (agermanidis) [inline docs/getting-started/model-101.md]:
```suggestion
We use the term model as a high-level representation of AI logic.  Models contain a series of parameters and soft rules that can be both decided by both humans and machine intelligence. For example, a model may itself learn how to recognize a face in a video or image. A researcher has given the model, images containing faces and over time, the model has learned what a face looks like. This is an example of a form of recognition. We often take recognition for granted, as we learned how to recognize people and inanimate objects as we learned as a child. But imagine how tricky it is for a computer to learn and recognize!
```

File: docs/getting-started/model-101.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+- **Transform** - a model transforms the input that is given to it. For example. If we apply stylisation to an image, so it looks like it's been painted by Vincent van Gogh.
+
+### ML Models in the world
+
+Now you know more about Models, what can models do? People have trained and built models for many different purposes. Below are some of the models we have detected:
+
+* Recognition: emotions, objects, people body positions, hot-dogs, spam and sounds
+* Generate: photorealistic faces, captions, streets, facades, voices and videos of airplanes morphing into cats
+* Detect: models to detect cancer, malaria and HIV
+
```
Comment (agermanidis) [inline docs/getting-started/model-101.md]: Maybe it'd be more clear if this section used the same categories with the ones above? Recognition seems to be very similar to detection here.

File: docs/getting-started/model-101.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+Some trained models are ethically dubious and raise important questions about the social impact of algorithms and biases in our society such as models used to screen job applicants or predict crime. People build models in different programming languages, with different frameworks and for different purposes.
+
+## Where to find models?
+
+On Runway, you can easily browse different types of models and search according to your needs. Please check out our guide on [browsing models on Runway](how-to/browse-model-directory).
+
+
+## Staff Picks: top models to try
+
+Currently on Runway we provided a short list of the teams favorite models to get started with! Below are some recommendations, or checkout our tutorial on [browsing models on Runway](how-to/browse-model-directory).
```
Comment (agermanidis) [inline docs/getting-started/model-101.md]:
```suggestion
Currently on Runway we provided a short list of the team's favorite models to get started with! Below are some recommendations, or checkout our tutorial on [browsing models on Runway](how-to/browse-model-directory).
```

**Pull request closed without merging**